### ISSUE TITLE
loki output multitenancy (X-Scope-OrgID)

### ISSUE THREAD

**Issue by yodog** [sipcapture/paStash#66]
hello.

i believe requests to the Loki API should include an HTTP header (X-Scope-OrgID) identifying the tenant for the request.

https://github.com/grafana/loki/blob/master/docs/operations/multi-tenancy.md

**Comment by lmangani** [comment]
That's right. The Loki output feature is currently tested in single-tenant mode - will add it once time allows and keep this open as feature request (or feel free to submit a PR)
